src/components/Button.js

```diff
@@ -41,15 +43,13 @@ module.exports = React.createClass({
 		};
 	},
 	render() {
+		var classes = this.props.sheet.classes;
```

Comment:
@nkbt 
1. thanks, will fix it as we agree on the rest.
1. You misunderstood the concept behind react-jss. It needs to hook into lifecycle callbacks to attach/detach style element. And keep selector engine fast, because sheets which are not in use are not in the render tree.